### DISCUSSION TITLE
Not an issue, just a command to install all deps.

### DISCUSSION THREAD

**Discussion by A-c0rN** [openbroadcaster/obplayer#44] [Show and tell]
If you're lazy, here's the command to install _**ALL DEPENDENCIES**_ for this application, including ones you might not need. 

Tweak according to your desires!

`sudo apt install ntp python3 python3-pycurl python3-openssl python3-apsw python3-magic python3-dateutil python3-requests python3-gi python3-gi-cairo gir1.2-gtk-3.0 gir1.2-gdkpixbuf-2.0 gir1.2-pango-1.0 python3-gst-1.0 gir1.2-gstreamer-1.0 gir1.2-gst-plugins-base-1.0 gir1.2-gst-rtsp-server-1.0 gstreamer1.0-tools gstreamer1.0-libav gstreamer1.0-alsa gstreamer1.0-pulseaudio gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly ffmpeg ubuntu-restricted-addons ubuntu-restricted-extras gstreamer1.0-vaapi mesa-vdpau-drivers espeak mbrola mbrola-en1 mbrola-us1 mbrola-us2 mbrola-us3 mbrola-fr1 mbrola-fr4 python3-serial cifs-utils && sudo pip3 install boto3 pulsectl pyrtlsdr inotify`

Thanks for reading, have a great day!
- Anastasia M.

**Comment by wdsmaduranga** [comment]
dear sir how to install this on on my vps can i give that commands
